Dockerfile

```diff
@@ -32,6 +34,16 @@ RUN mkdir tessdata && cd tessdata \
     && find . -name '*.traineddata' -maxdepth 2 -exec cp {} /usr/share/tessdata \; \
     && cd .. && rm -r tessdata
 
+RUN mkdir h2orestart && cd h2orestart \
+    && H2ORESTART_FILENAME=h2orestart.oxt \
+    && H2ORESTART_VERSION="v0.5.5" \
```

Comment:
I thought that since the checksum is also hard-coded. It might not needed. But I guess your recommendation is far better. I will change to the way to get tessdata.

Reply:
Thanks!

Reply:
I just changed!